cli/azd/cmd/testdata/TestUsage-azd-pipeline-config.snap

```diff
@@ -7,13 +7,13 @@ Usage
   azd pipeline config [flags]
 
 Flags
-        --auth-type string      	: The authentication type used between the pipeline provider and Azure for deployment (Only valid for GitHub provider). Valid values: federated, client-credentials.
-    -e, --environment string    	: The name of the environment to use.
-    -h, --help                  	: Gets help for config.
-        --principal-name string 	: The name of the service principal to use to grant access to Azure resources as part of the pipeline.
-        --principal-role string 	: The role to assign to the service principal.
-        --provider string       	: The pipeline provider to use (github for Github Actions and azdo for Azure Pipelines).
-        --remote-name string    	: The name of the git remote to configure the pipeline to run on.
+        --auth-type string           	: The authentication type used between the pipeline provider and Azure for deployment (Only valid for GitHub provider). Valid values: federated, client-credentials.
+    -e, --environment string         	: The name of the environment to use.
+    -h, --help                       	: Gets help for config.
+        --principal-name string      	: The name of the service principal to use to grant access to Azure resources as part of the pipeline.
+        --principal-role stringArray 	: The roles to assign to the service principal.
```

Comment:
Maybe we can add something like: (by default, the service principal is granted Contributor and User Access Administrator access) so folks understand what we we default to?